src/Apps/Collect/Components/ArtworkGrid/index.tsx

```diff
@@ -134,6 +136,29 @@ class Filter extends Component<Props> {
                                   {!hideTopBorder && <Separator mb={1} />}
 
                                   {this.renderWaysToBuy(filters, mediator)}
+
+                                  <div>
+                                    <Range
```

Comment:
Still need to update text above the slider to show you the range you've selected